### ISSUE TITLE
Use of SuperUser may not be documented

### ISSUE THREAD

**Issue by arihantdaga** [emqtt/emqttd#1605]
#### Environment

- OS: 
- Erlang/OTP: 
- EMQ: 
2.3.6

#### Description
Hello, I read all the docs, also saw that through mongo_db queries we can set `is_superuser: true/false ` for a user. But i did not understand what is the use of super_user and what special permissions it will have ? will a super user be able to pub sub to any topic ? or something else.. ?
Can somebody clear this confusion.. ?

**Comment by terry-xiaoyu** [comment]
@arihantdaga Yes, the super user is allowed to sub/pub to any topic. I think this user is useful for testing.

**Comment by arihantdaga** [comment]
@terry-xiaoyu Thank you.
